pkg/gcp/disk.py

```diff
@@ -133,26 +133,37 @@ async def start_image_creation(project, hostname):
    return True
 
 
-async def start_cloning(project, hostname):
+async def start_cloning(project):
```

Comment:
cloning should happen 1 host at a time, from frontend user clicks clone button per host.